server/ops/docker/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+# Specify svgbob_cli via Cargo.toml so that Dependabot can help with updates
+
+[package]
+name = "kroki_svgbob_cli"
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "svgbob_cli"
+path = "/home/rust/.cargo/bin/svgbob"
```

Comment:
I don't understand why we need to specify a `path` since the binary ends up in `/home/rust/.cargo/registry/index/*/.cache/sv/gb/svgbob`?